discovery-infra/start_discovery.py

```diff
@@ -300,6 +313,17 @@ def main():
         nodes_flow(client, cluster_name, cluster)
 
 
+def str2bool(v):
```

Comment:
it is not more capable :) 
use at least distutils.util.strtobool
but i think it is not needed